modules/core/shortcodes.php

```diff
@@ -440,6 +440,8 @@ class="orbisius_support_tickets_data_message form-control orbisius_support_ticke
                                           rows="<?php echo $row_num; ?>"><?php esc_attr_e( $data['message'] ); ?></textarea>
                             </div>
                         </div>
+                        
+                        <?php do_action( 'orbisius_support_tickets_action_submit_ticket_form_add_fields', $ctx ); ?>
```

Comment:
Can you please rename: orbisius_support_tickets_action_submit_ticket_form_add_fields 
to
orbisius_support_tickets_action_submit_ticket_form_before_submit_button

reason: is that this hook can be used to add more stuff and not just fields.

Reply:
it's great that you're following the way the other hooks are named 😄

Reply:
Done